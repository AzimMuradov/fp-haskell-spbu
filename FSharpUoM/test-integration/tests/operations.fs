let x = 5;
let y = 21;
x + y;; // Right: "26"

let x = 5;
let y = 21;
x - y;; // Right: "(-16)"

let x = 5;
let y = 21;
x * y;; // Right: "105"

let x = 5;
let y = 21;
y / x;; // Right: "4"

let x = 5;
let y = 21;
y % x;; // Right: "1"

let x = 2;
let y = 5;
x ** y;; // Right: "It is not possible to apply this operation between 'int' and 'int'"

let x = 5.0;
let y = 2;
x ** y;; // Right: "25.0"

let x = 5;
let y = 21;
x = y;; // Right: "false"

let x = 5;
let y = 21;
x <> y;; // Right: "true"

let x = 5;
let y = 21;
x < y;; // Right: "true"

let x = 5;
let y = 21;
x <= y;; // Right: "true"

let x = 5;
let y = 21;
x > y;; // Right: "false"

let x = 21;
let y = 21;
x >= y;; // Right: "true"