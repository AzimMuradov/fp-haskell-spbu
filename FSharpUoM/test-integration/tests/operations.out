val it: int = 26
val it: int = -16
val it: int = 105
val it: int = 4
val it: int = 1
It is not possible to apply this operation between 'int' and 'int'
val it: double = 25.0
val it: bool = False
val it: bool = True
val it: bool = True
val it: bool = True
val it: bool = False
val it: bool = True
